docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc

```diff
@@ -1,26 +1,56 @@
-= SBOMs
+= Inspecting SBOMs
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
```

Comment:
Am I incorrect that it's `{Product_Name}`

Reply:
`In {Product_Name}, **a** SBOM lists all the software libraries that a component uses.`

I don't think we need to hyperlink to the component doc. I think linking would honestly make users leave and never return to this page. 

If you'd like, we can include an `Additional resources` section at the end where we can link to the glossary.

Reply:
IBM SG discourages the use of `may` 

What do you think of: 

`Those libraries can enable specific functionality or facilitate development.` 

I think `serve some other purpose` is a little vague, so I'd recommend removing it.

Reply:
It is {ProductName} :+1:  and thanks for the minimalist suggestions!